src/apps/AppCenter/InstalledApps/RepoVersions.js

```diff
@@ -63,11 +63,11 @@ const RepoVersions = ({ animate, repo: { currentVersion, versions } }) => (
               <Td>
                 {version}{' '}
                 {version === currentVersion.version && (
-                  <Badge.Identity
+                  <Tag mode="identity"
```

Comment:
We probably want to use a different mode now?

Reply:
Which one? or what are your complete thoughts on this?

Reply:
I guess I'm just generally confused with the direction that `mode="app"` and `mode="identity"` are going.

From what I understand, both of these are going to be removed for `AppBadge` and `IdentityBadge`, and are going to just render a vanilla `Tag` anyway. If that's the case, shouldn't we either:

1. Use `Tag` directly
1. Use a different mode

Reply:
I share the confusion but in the meantime this can make it so we have one source of truth: `Tag` and it'll help us update when needed.